### PR TITLE
fixed issue #65 'Not compatible with node.js v4' (tested on 4.8.7)

### PR DESCRIPTION
**Pull request by graham-h**
Had the same issue as https://github.com/google/closure-compiler-js/issues/65 - the only incompatibilities appear to be in cmd.js.

### CODE REVIEW

**Comment by MatrixFrog** [comment]
If compatibility with node 4 is important then let's make sure Travis is running tests on node 4. Otherwise it will surely break again.

**Comment by graham-h** [comment]
Quite right.  It matters to _me_, and presumably to yeyingxian - beyond that I'm just treating the README as a spec, and that says "supports ... Node v4 LTS".

Not familiar with Travis, so I hope my change is correct.

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 language: node_js
 
 node_js:
- - stable
+ - lts/argon
```
Comment (MatrixFrog) [inline .travis.yml]: I think you can add multiple entries here right? So let's do

 - lts/argon
 - stable

so that it runs on both.

Comment (graham-h) [reply .travis.yml]: One way to find out...

**Pull request merged**